base/mpfr.jl

```diff
@@ -681,6 +700,11 @@ cmp(x::CdoubleMax, y::BigFloat) = -cmp(y,x)
 
 signbit(x::BigFloat) = ccall((:mpfr_signbit, :libmpfr), Int32, (Ptr{BigFloat},), &x) != 0
 
+"""
+    precision(BigFloat)
```

Comment:
`precision` docstring, is attached to the signature for `precision(x::BigFloat)` but the docstring content is describing the `precision(::Type{BigFloat})` docstring right below